test/sagemaker_tests/tensorflow/tensorflow1_training/requirements.txt

```diff
@@ -20,5 +20,4 @@ botocore
 requests-mock
 awscli
 fabric==2.5.0
-sagemaker-experiments==0.1.7
```

Comment:
Why remove this?

Reply:
sagemaker-experiments needs to be installed only for py3 images. so I moved this to expliclty install for py3 images